data/vibe/data/json.d

```diff
@@ -1925,6 +1926,17 @@ struct JsonStringSerializer(R, bool pretty = false)
 	this(R range)
 	{
 		m_range = range;
+		static if (isOutputRange!(R, char))
+		{
+			dg = (scope const(char)[] data)
+			{
+				static if (__traits(compiles, m_range ~= data))
+					m_range ~= data;
```

Comment:
in case the OutputRange supports more than just the put method(like with Appender) and operator~ is more optimized

Reply:
It would be better to have this as a comment in the code ;)